sourmash/__main__.py

```diff
@@ -73,8 +85,19 @@ def main():
         print(usage)
         sys.exit(1)
 
-    cmd = commands.get(args.command)
-    cmd(sys.argv[2:])
+    if args.command in loadable_cmds:
+        parser = argparse.ArgumentParser(
+            description='work with compressed sequence representations')
+        subparsers = parser.add_subparsers()
+        subp = subparsers.add_parser(args.command)
+        commands[args.command].add_args(subp)
+        subp.set_defaults(cmd=commands[args.command])
+        args = parser.parse_args()
+        return args.cmd(**vars(args))
+    else:
+        cmd = commands.get(args.command)
+        return cmd(sys.argv[2:])
+
```

Comment:
Once all commands are converted we can remove this `if` block and keep only the first part (the `else` is the current implementation for subcommands)